streams/src/main/java/org/apache/kafka/streams/processor/internals/InternalTopologyBuilder.java

```diff
@@ -1148,6 +1148,17 @@ private static Pattern buildPattern(final Collection<String> sourceTopics,
         return results;
     }
 
+    public List<String> sourceTopicsForStore(final String storeName) {
+        final List<String> results = new ArrayList<>();
+        for (final Map.Entry<String, Set<String>> entry : stateStoreNameToSourceTopics.entrySet()) {
+            if (entry.getKey().equals(storeName)) {
+                results.addAll(maybeDecorateInternalSourceTopics(entry.getValue()));
+                return results;
+            }
+        }
+        return null;
```

Comment:
any reason we can't use an empty list for the sentinel?

Reply:
Done.